src/EnergyPlus/DataZoneEquipment.cc

```diff
@@ -681,6 +701,9 @@ namespace DataZoneEquipment {
 
 					}}
 				}
+				// If there are two or more air terminals in a zone, then set minimum iterations to number of air terminals
+				DataHVACGlobals::MinAirLoopIterationsAfterFirst = max ( MinAirLoopIterationsAfterFirst, countAirTermsInZone );
```

Comment:
Why? If there is an air loop and zone equipment or multiple zone equipment we don't set a min iteration, so why now? If there are 3 air loops will min iteration be 3?

Reply:
The prompt for the extra iteration(s) was a test file with 1 zone and 2 furnace airloops, both sized to 100%.  I expected the first airloop to pick up all of the load and the second one to remain off.  Instead, they both ran to meet the full load, then they were off the next time step and back and forth like that.  The sequenced loads are all initialized to the full output required for the timestep.  The existing code, sets the sequenced loads to full, runs the airloops for firsthvaciteration, then runs the zone equpment, then resets the sequenced loads to full, then runs the airloops, so they all try to meet the full load.  Then the zone equipment runs, and the sequenced load is reduced for the second equipment type, but we're done.  So, adding another iteration fixes the problem.  I haven't made a test file yet with 3 airloops, but I expect that an additional iteration will be required there (1st iteration, simulate furnace 1 correctly, 2nd iteration, gets the 2nd furnace right, then the 3rd iteration gets the 3rd one right).

This example was all for the SequencedLoad distrbution type where each successive piece of equipment attempts to meet the remaining load.  For the other load distribution types which are preset at the get-go, the extra iterations may not be necessary.

Reply:
OK, so RemainingOutputRequired for each air loop doesn't get decremented until the TUs are simulated. Once that happens SequencedOutputToCoolingSP for air loop 2 should be 0. So to test this break on the second air loop on the second iteration and check the values of the sequenced load variables.